external/dynamo/.env.example

```
@@ -13,18 +13,32 @@
+
+# Set repository directory (for Thompson Sampling router)
+export DYNAMO_REPO_DIR="/path/to/NeMo-Agent-Toolkit/external/dynamo"
 
 # =============================================================================
 # OPTIONAL VARIABLES - GPU Configuration
 # =============================================================================
 
 # GPU device IDs for unified mode (comma-separated)
 # Default: 0,1,2,3
-DYNAMO_GPU_DEVICES="0,1,2,3"
+export DYNAMO_GPU_DEVICES="0,1,2,3"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)